sycl/doc/UsersManual.md

```diff
@@ -1,63 +1,116 @@
 # Users Manual
 
-The DPC++ Compiler contains many options to generate the desired binaries for
-your application.
+This is the list of SYCL specific options supported by compiler and some
+examples.
 
-## SYCL specific command line options
+## Generic options
 
 **`-fsycl`**
 
-    General enabling option for SYCL compilation mode.  This option enables
-    offload compilation for a given target using the `-fsycl-targets` option.
-    When the `-fsycl-targets` option is not provided, the default triple is
-    `spir64-unknown-unknown-sycldevice`.
+    General enabling option for SYCL compilation and linking mode. List of
+    targets can be specified with `-fsycl-targets`. This is fundamental option
+    for any SYCL compilation. All other SYCL specific options require it.
 
-**`-fsycl-targets=<value>`**
+**`-fsycl-targets=<T1>[,...,<Tn>]`**
 
-    A comma separated list of triples to specify the device target(s) to
-    generate code for.  This option is only valid when used with `-fsycl`.
+    Tells the compiler to generate code for specified devices. T is a target
+    triple device name. You can specify more than one target, comma separated.
+    The following triples are supported:
+    * spir64-unknown-unknown-sycldevice - this is the default generic SPIR-V
+      target;
+    * spir64_x86_64-unknown-unknown-sycldevice - generate code ahead of time
+      for x86_64 CPUs;
+    * spir64_fpga-unknown-unknown-sycldevice - generate code ahead of time for
+      Intel FPGA;
+    * spir64_gen-unknown-unknown-sycldevice - generate code ahead of time for
+      Intel Processor Graphics;
+    * nvptx64-nvidia-cuda-sycldevice - generate code ahead of time for CUDA
+      target;
 
-### Target toolchain options.
+## Language options
 
-**`-Xsycl-target-backend=<triple> <arg>`**
+**`-sycl-std=<value>`** [EXPERIMENTAL]
 
-    Pass <arg> to the SYCL based backend identified by <triple>.
+    SYCL language standard to compile for. Possible values:
+    * 121 - SYCL 1.2.1
+    * 2020 - SYCL 2020
+    It doesn't guarantee specific standard compliance, but some selected
+    compiler features change behavior.
+    It is under development and not recommended to use in production
+    environment.
+    Default value is 2020.
 
-**`-Xsycl-target-backend <arg>`**
+**`-f[no-]sycl-unnamed-lambda`**
 
-    Pass <arg> to the SYCL based target backend.
+    Enables/Disables unnamed SYCL lambda kernels support.
+    Enabled by default.
```

Comment:
I don't think it's enabled by default. @mdtoguchi, am I right?

Reply:
I was thinking it's on under sycl-std=2020, but I could be wrong. @mdtoguchi?

Reply:
This is not what I see in the source code: https://github.com/intel/llvm/blob/sycl/clang/lib/Driver/ToolChains/Clang.cpp#L6589.

Reply:
Right, this is not enabled by default.

Reply:
Thanks. Fixed.